tests/tests/rap_noah_sfcdiff_unified_ugwp

```diff
@@ -93,3 +93,5 @@ export DO_TOFD=.true.
 export GWD_OPT=2
 export DO_GSL_DRAG_SS=.true.
 export DO_GSL_DRAG_TOFD=.true.
+
+export DIAG_FLUX=.true.
```

Comment:
@SamuelTrahanNOAA this suite does not use MYNN surface layer scheme, but uses sfc_diff.f. The diag_flux should be default .false.

Reply:
Corrected.

Reply:
That test is not run, so it won't pass or fail no matter what I do. I'll remove DIAG_FLUX.